backend/fms_core/viewsets/container.py

```diff
@@ -106,7 +106,12 @@ def create(self, request):
                 raise ValidationError({"barcode": f"Container with name {container['barcode']} already exists. Missing container name, barcode will replace container name."})
 
             container = remove_empty_str_from_dict(container)
-            container_obj, errors, warnings = create_container(barcode=container['barcode'], kind=container['kind'], name=container['name'], coordinates=container['coordinate'], container_parent=container['location'], creation_comment=container['comment'])
+            if container['location']:
+                try:
+                    container_parent = Container.objects.get(id=container['location'])
```

Comment:
i believe `container_parent` needs to be declared beforehand (maybe set to `None`).
Otherwise an error like this can occur:
```python
Traceback (most recent call last):
  File "/Users/nafizislam/projects/freezeman/backend/fms_core/viewsets/container.py", line 114, in create
    container_obj, errors, warnings = create_container(barcode=container['barcode'], kind=container['kind'], name=container['name'], coordinates=container['coordinate'], container_parent=container_parent, creation_comment=container['comment'])
                                                                                                                                                                                           ^^^^^^^^^^^^^^^^

During handling of the above exception (cannot access local variable 'container_parent' where it is not associated with a value), another exception occurred:
  File "/Users/nafizislam/projects/freezeman/backend/env/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nafizislam/projects/freezeman/backend/fms_core/viewsets/container.py", line 118, in create
    raise ValidationError(errors)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```